public/docs/v2/components/paths/term/group-param-id.yaml

```
@@ -1,5 +1,8 @@
 get:
   $ref: './get-one.yaml'
 
+patch:
+  $ref: './update.yaml'
+
 delete:
   $ref: './delete.yaml'
```

apenas adicionar uma linha no final do arquivo para manter o padrão
```suggestion
  $ref: './delete.yaml'
  
```